### PR TITLE
feat : Crear gráfica de resultados de votación #9

### PR DESCRIPTION
**Pull request by adrferfer**
Creada gráfica para mostrar el resultado de las votaciones

### CODE REVIEW

File: decide/booth/static/js/grafica_visualizer.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          x: {
+            title: {
+              display: true,
+              text: 'Opciones'
+            }
+          },
+          y: {
+            title: {
+              display: true,
+              text: 'Número de votos'
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:74]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
              text: "Número de votos"
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+for(i=0; i< votos.length ; i++){
+    if(votos[i] != 0 && totalVotos != 0){
+        porVotosPorOpcion.push((votos[i]/totalVotos)*100);
+    } else {
+        porVotosPorOpcion.push(votos[i]);
+    }
+}
+
+var config1 = new Chart(ctx1, {
+    type: 'bar',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:26]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
    type: "bar",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                'rgba(54, 162, 235, 0.4)',
+                'rgba(153, 102, 255, 0.4)',
+                'rgba(201, 203, 207, 0.4)'
+              ],
+              borderColor: [
+                'rgb(255, 99, 132)',
+                'rgb(255, 159, 64)',
+                'rgb(255, 205, 86)',
+                'rgb(75, 192, 192)',
+                'rgb(54, 162, 235)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:45]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(54, 162, 235)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        datasets: [{
+            label: 'Resultados de la votación',
+            data: votos,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.4)',
+                'rgba(255, 159, 64, 0.4)',
+                'rgba(255, 205, 86, 0.4)',
+                'rgba(75, 192, 192, 0.4)',
+                'rgba(54, 162, 235, 0.4)',
+                'rgba(153, 102, 255, 0.4)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:37]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgba(153, 102, 255, 0.4)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                'rgba(201, 203, 207, 0.4)'
+              ],
+              borderColor: [
+                'rgb(255, 99, 132)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:41]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(255, 99, 132)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+            title: {
+              display: true,
+              text: 'Resultados de la votación'
+            }
+        },
+        scales: {
+          x: {
+            title: {
+              display: true,
+              text: 'Opciones'
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:68]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
              text: "Opciones"
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+var config1 = new Chart(ctx1, {
+    type: 'bar',
+    data: {
+        datasets: [{
+            label: 'Resultados de la votación',
+            data: votos,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.4)',
+                'rgba(255, 159, 64, 0.4)',
+                'rgba(255, 205, 86, 0.4)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:34]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgba(255, 205, 86, 0.4)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+                'rgba(75, 192, 192, 0.4)',
+                'rgba(54, 162, 235, 0.4)',
+                'rgba(153, 102, 255, 0.4)',
+                'rgba(201, 203, 207, 0.4)'
+              ],
+              borderColor: [
+                'rgb(255, 99, 132)',
+                'rgb(255, 159, 64)',
+                'rgb(255, 205, 86)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:43]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(255, 205, 86)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+                'rgba(255, 159, 64, 0.4)',
+                'rgba(255, 205, 86, 0.4)',
+                'rgba(75, 192, 192, 0.4)',
+                'rgba(54, 162, 235, 0.4)',
+                'rgba(153, 102, 255, 0.4)',
+                'rgba(201, 203, 207, 0.4)'
+              ],
+              borderColor: [
+                'rgb(255, 99, 132)',
+                'rgb(255, 159, 64)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:42]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(255, 159, 64)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 10]
```diff
@@ -0,0 +1,124 @@
+var ctx1 = document.getElementById('bar-resultados').getContext('2d');
+var ctx2 = document.getElementById('pie-porcentaje').getContext('2d');
+
+var votacion = JSON.parse(document.getElementById('votacion').value);
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
var votacion = JSON.parse(document.getElementById("votacion").value);
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 11]
```diff
@@ -0,0 +1,124 @@
+var ctx1 = document.getElementById('bar-resultados').getContext('2d');
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
var ctx1 = document.getElementById("bar-resultados").getContext('2d');
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+    type: 'bar',
+    data: {
+        datasets: [{
+            label: 'Resultados de la votación',
+            data: votos,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.4)',
+                'rgba(255, 159, 64, 0.4)',
+                'rgba(255, 205, 86, 0.4)',
+                'rgba(75, 192, 192, 0.4)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:35]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgba(75, 192, 192, 0.4)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+    },
+    options: {
+        responsive: true,
+        plugins: {
+            legend: {
+              position: 'top',
+            },
+            title: {
+              display: true,
+              text: 'Resultados de la votación'
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:61]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
              text: "Resultados de la votación"
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+            label: 'Resultados de la votación',
+            data: votos,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.4)',
+                'rgba(255, 159, 64, 0.4)',
+                'rgba(255, 205, 86, 0.4)',
+                'rgba(75, 192, 192, 0.4)',
+                'rgba(54, 162, 235, 0.4)',
+                'rgba(153, 102, 255, 0.4)',
+                'rgba(201, 203, 207, 0.4)'
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:38]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgba(201, 203, 207, 0.4)"
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+              ],
+              borderColor: [
+                'rgb(255, 99, 132)',
+                'rgb(255, 159, 64)',
+                'rgb(255, 205, 86)',
+                'rgb(75, 192, 192)',
+                'rgb(54, 162, 235)',
+                'rgb(153, 102, 255)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:46]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(153, 102, 255)",
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 10 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
var votacion = JSON.parse(document.getElementById("votacion").value);
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+                'rgba(75, 192, 192, 0.4)',
+                'rgba(54, 162, 235, 0.4)',
+                'rgba(153, 102, 255, 0.4)',
+                'rgba(201, 203, 207, 0.4)'
+              ],
+              borderColor: [
+                'rgb(255, 99, 132)',
+                'rgb(255, 159, 64)',
+                'rgb(255, 205, 86)',
+                'rgb(75, 192, 192)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:44]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(75, 192, 192)",
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 1 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:74]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
              text: "Número de votos"
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 13 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:61]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
              text: "Resultados de la votación"
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+    data: {
+        datasets: [{
+            label: 'Resultados de la votación',
+            data: votos,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.4)',
+                'rgba(255, 159, 64, 0.4)',
+                'rgba(255, 205, 86, 0.4)',
+                'rgba(75, 192, 192, 0.4)',
+                'rgba(54, 162, 235, 0.4)',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:36]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgba(54, 162, 235, 0.4)",
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 5 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:41]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(255, 99, 132)",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+          }
+        }
+    }
+});
+
+var config2 = new Chart(ctx2, {
+    type: 'pie',
+    data: {
+        datasets: [{
+            label: 'Porcentaje de votos por opción',
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:88]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
            label: "Porcentaje de votos por opción",
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 11 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
var ctx1 = document.getElementById("bar-resultados").getContext('2d');
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 6 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:68]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
              text: "Opciones"
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+    },
+    options: {
+        responsive: true,
+        plugins: {
+            legend: {
+              position: 'top',
+            },
+            title: {
+              display: true,
+              text: 'Porcentaje de votos por opción'
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:120]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
              text: "Porcentaje de votos por opción"
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 20]
```diff
@@ -0,0 +1,124 @@
+var ctx1 = document.getElementById('bar-resultados').getContext('2d');
+var ctx2 = document.getElementById('pie-porcentaje').getContext('2d');
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
var ctx2 = document.getElementById("pie-porcentaje").getContext('2d');
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 4 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:37]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgba(153, 102, 255, 0.4)",
```

File: decide/booth/static/js/grafica_visualizer.js (hunk 2 above)
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:26]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
    type: "bar",
```

File: decide/booth/static/js/grafica_visualizer.js [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+                'rgba(201, 203, 207, 0.4)'
+              ],
+              borderColor: [
+                'rgb(255, 99, 132)',
+                'rgb(255, 159, 64)',
+                'rgb(255, 205, 86)',
+                'rgb(75, 192, 192)',
+                'rgb(54, 162, 235)',
+                'rgb(153, 102, 255)',
+                'rgb(201, 203, 207)'
```
Comment (marmolpen3) [inline decide/booth/static/js/grafica_visualizer.js:47]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgb(201, 203, 207)"
```

**Review by Albert0led0 — APPROVED** [review]
Todo perfecto

**Review by iarrans — APPROVED** [review]
Todo parece correcto, a codacy también se lo parece

**Pull request merged**